vkconfig/mainwindow.ui

```diff
@@ -100,119 +100,127 @@
-              <width>401</width>
-              <height>18</height>
-             </rect>
-            </property>
-            <property name="font">
-             <font>
-              <family>Arial</family>
-              <pointsize>10</pointsize>
-             </font>
-            </property>
-            <property name="text">
-             <string>Apply only to the selected list of Vulkan applications</string>
-            </property>
-           </widget>
-           <widget class="QCheckBox" name="checkBoxPersistent">
-            <property name="geometry">
-             <rect>
-              <x>30</x>
-              <y>105</y>
-              <width>371</width>
-              <height>19</height>
-             </rect>
-            </property>
-            <property name="font">
-             <font>
-              <family>Arial</family>
-              <pointsize>10</pointsize>
-             </font>
-            </property>
-            <property name="text">
-             <string>Make layers override persistent on exit</string>
-            </property>
-           </widget>
-           <widget class="QRadioButton" name="radioFully">
-            <property name="geometry">
-             <rect>
-              <x>14</x>
-              <y>31</y>
-              <width>331</width>
-              <height>20</height>
-             </rect>
-            </property>
-            <property name="font">
-             <font>
-              <family>Arial</family>
-              <pointsize>10</pointsize>
-             </font>
-            </property>
-            <property name="text">
-             <string>Fully controlled by the Vulkan applications</string>
-            </property>
-           </widget>
-           <widget class="QPushButton" name="pushButtonAppList">
-            <property name="geometry">
-             <rect>
-              <x>440</x>
-              <y>80</y>
-              <width>80</width>
-              <height>26</height>
-             </rect>
-            </property>
-            <property name="sizePolicy">
-             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
-              <horstretch>0</horstretch>
-              <verstretch>0</verstretch>
-             </sizepolicy>
-            </property>
-            <property name="minimumSize">
-             <size>
-              <width>80</width>
-              <height>26</height>
-             </size>
-            </property>
-            <property name="maximumSize">
-             <size>
-              <width>100</width>
-              <height>16777215</height>
-             </size>
-            </property>
-            <property name="font">
-             <font>
-              <family>Arial</family>
-              <pointsize>10</pointsize>
-             </font>
-            </property>
-            <property name="text">
-             <string>Edit...</string>
-            </property>
-           </widget>
+           <layout class="QGridLayout" name="gridLayout_7">
```

Comment:
all of these changes are due to the addition of a wrapping `QGridLayout`, yes?  I didn't scrutinize the details since they seem to be changes in whitespace only.  (If there was another substantive change, point me at it and I'll look.)

Reply:
Yes, a new layout had to be added for spacing reasons. Layouts are wonderful, it's taken me a while to master their use though.